### PR TITLE
formatting and python code examples for HDF5 files.

### PR DESCRIPTION
**Pull request by nkicg6**
related to ilastik/ilastik/issues/2251.  Reformatting and code examples for HDF5 files.

### CODE REVIEW

**Comment by nkicg6** [comment]
@k-dominik let me know if this is too much detail. I wanted to expand the HDF5 export documentation you added before adding object measurement documentation (discussed here ilastik/ilastik/issues/2251). 
It might make sense to add object measurement documentation  to a new file that we can link to.

File: documentation/objects/objects.md [hunk 1]
```diff
@@ -223,14 +223,59 @@ In this new window there are three vertical tabs:
...
+#('object_id', 'timestep', 'Predicted Class' ...)
+```
+
+Individual fields can be accessed by name:
+
+```python
+data['table']['Mean Intensity']
+# array([1551.2393, 1420.5,...], dtype=float32)
+```
+Returning a numpy array of all the detected objects. You can access an individual object's measurements using an index:
```
Comment (k-dominik) [inline documentation/objects/objects.md:272]:
```suggestion
Returning a numpy array of `Mean Intensity` values for all the detected objects.
You can access an individual object's measurements using an index:
```

**Comment by nkicg6** [comment]
Thanks @k-dominik, I like HDF5 a lot now that I understand how to interact with it, so I hope this helps others. I incorporated your suggestions, let me know if there are any problems!

File: documentation/objects/objects.md [hunk 2]
```diff
@@ -223,14 +223,59 @@ In this new window there are three vertical tabs:
...
+# <KeysViewHDF5 ["labeling", "raw"]>
+data["images"]["1"]["labeling"]
+# <HDF5 dataset "labeling": shape (28, 26, 13), type "|u1">
+# stored as a numpy array, access the data using numpy indexing:
+data["images"]["1"]["labeling"][:]
+# image data...
+```
+
+The measurements `table` is saved as a [numpy structured array](https://numpy.org/doc/stable/user/basics.rec.html?highlight=structured#module-numpy.doc.structured_arrays) and holds the selected feature values for each object.
+The "columns" are saved as dtypes (you can see all column names in your table:
```
Comment (k-dominik) [inline documentation/objects/objects.md:259]:
```suggestion
The "columns" are saved as dtypes (you can see all column names in your table):
```

Comment (k-dominik) [reply documentation/objects/objects.md:259]: Hey this looks good, just a closing brace and this is done :)

Comment (nkicg6) [reply documentation/objects/objects.md]: oops thanks just fixed with 12c9c70!

**Comment by nkicg6** [comment]
cool thanks for your help!

**Comment by k-dominik** [comment]
Thank you very much for you contribution!! Highly appreciated on our side :)

**Pull request merged**